src/main/java/seedu/address/model/event/Description.java

```diff
@@ -0,0 +1,47 @@
+package seedu.address.model.event;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
```

Comment:
Add class level javadocs to tell us about the class

Reply:
Done-by Sol